plugin.cpp

```diff
@@ -38,6 +38,8 @@ bool CellularAutomata1D::init()
 
     // determines which rule to use
     m_rule = attr("rule").toInt();
+    m_binrule = QString::number(m_rule, 2);
+    m_binrule.prepend(QString("0").repeated(8 - m_binrule.length()));
```

Comment:
That's an elegant solution using QString :clap: 
However, I think we should just use a std::bitset here.
``` c++
m_binrule = std::bitset<8>(attr("rule").toInt());
```
 Please, fix the type of `m_binrule` to `std::bitset<8> m_binrule` in the `plugin.h` file. Also, we can now remove the `m_rule` member.